CMakeLists.txt

```diff
@@ -247,11 +248,18 @@ target_link_libraries(openomf_core PRIVATE
     openomf::confuse
     openomf::enet
     openomf::epoxy
-    openomf::png
     openomf::SDL2
     openomf::SDL2_mixer
     openomf::xmp
 )
+
+if (USE_LIBPNG)
+    target_link_libraries(openomf_core PUBLIC
+        openomf::png
+    )
+    add_definitions(-DUSE_LIBPNG=1)
```

Comment:
Existing C macros for library detection are of the form: MINIUPNPC_FOUND, ARGTABLE3_FOUND, and NATPMP_FOUND.

Perhaps this macro should be PNG_FOUND instead of USE_LIBPNG?

Reply:
I don't mind, that works too. I used USE_ prefix, since I think of it as a decision to use/not use libpng. FOUND has a slightly different connotation, but it works too.